### PR TITLE
versioneer, documentation things, etc

### PR DESCRIPTION
**Pull request by wasade**

### CODE REVIEW

**Comment by wasade** [comment]
@dpear I *think* we can delete `taxonomy` from `plugin_setup.py` as we are going to require the input table be collapsed, so I didn't add a line of documentation for that import

note most of this PR is for `versioneer` which standardizes the handling of version strings.

**Comment by wasade** [comment]
...CI will fail too as we haven't gone through resolving pep8, which we can do later

**Comment by dpear** [comment]
Should we rename the `control` variable which takes on values of (`atcc`, `zymobiomics`, `classic`) to `environment` or `community` ?? Might just be me but `control` and `positive_control_value` seem very easy to mix up.

**Comment by wasade** [comment]
`control_type`?

File: q2_katharoseq/tests/test_method.py [hunk 1]
```diff
@@ -8,8 +8,8 @@
 from qiime2 import NumericMetadataColumn
 
 from q2_katharoseq import read_count_threshold, estimating_biomass
-from q2_katharoseq import allosteric_sigmoid
-from q2_katharoseq import get_threshold
+from q2_katharoseq._methods import allosteric_sigmoid
```
Comment (antgonza) [inline q2_katharoseq/tests/test_method.py:11]: should these be added to `__init__.py` so they can be imported directly?

File: .github/workflows/python-package-conda.yml [hunk 2]
```diff
@@ -10,40 +10,44 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Python 3.9
```
Comment (antgonza) [inline .github/workflows/python-package-conda.yml:13]: Just to confirm, fine to remove 3.9?

**Review by antgonza — COMMENTED** [review]
Looks good, just a couple of question on the PR body + (a) want to confirm [that these warnings](https://github.com/biocore/q2-katharoseq/runs/5293611511?check_suite_focus=true#step:8:16) are fine to ignore or do they need to be fixed?; and (b) the README is missing how to run `estimating_biomass` from the CLI, is that expected?

File: q2_katharoseq/tests/test_method.py (hunk 1 above)
Comment (wasade) [reply q2_katharoseq/tests/test_method.py:11]: they aren't intended to be used generally so import from private is fine

File: .github/workflows/python-package-conda.yml (hunk 2 above)
Comment (wasade) [reply .github/workflows/python-package-conda.yml:13]: qiime2 is 3.8

**Comment by wasade** [comment]
Thanks! warnings are mostly sklearn but the pandas copy is probably worth fixing. 

the prior PR didn't include connecting estimating biomass to the API so there is nothing yet that can be added to the readme

**Pull request merged**